mac

```diff
@@ -138,40 +133,43 @@ if brew list | grep -Fq brew-cask; then
 fi
 
 fancy_echo "Updating Homebrew formulae ..."
+brew_tap 'thoughtbot/formulae'
 brew update
-
-brew_install_or_upgrade 'zsh'
-brew_install_or_upgrade 'git'
-brew_install_or_upgrade 'postgres'
-brew_launchctl_restart 'postgresql'
-brew_install_or_upgrade 'redis'
-brew_launchctl_restart 'redis'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'vim'
 brew_install_or_upgrade 'ctags'
-brew_install_or_upgrade 'tmux'
-brew_install_or_upgrade 'reattach-to-user-namespace'
-brew_install_or_upgrade 'imagemagick'
-brew_install_or_upgrade 'qt'
+brew_install_or_upgrade 'git'
+brew_install_or_upgrade 'heroku-toolbelt'
 brew_install_or_upgrade 'hub'
+brew_install_or_upgrade 'imagemagick'
+brew_install_or_upgrade 'libyaml'
 brew_install_or_upgrade 'node'
-
+brew_install_or_upgrade 'postgres'
+brew_install_or_upgrade 'qt'
 brew_install_or_upgrade 'rbenv'
+brew_install_or_upgrade 'rcm'
+brew_install_or_upgrade 'reattach-to-user-namespace'
+brew_install_or_upgrade 'redis'
 brew_install_or_upgrade 'ruby-build'
+brew_install_or_upgrade 'the_silver_searcher'
+brew_install_or_upgrade 'tmux'
+brew_install_or_upgrade 'vim'
+brew_install_or_upgrade 'zsh'
 
-# shellcheck disable=SC2016
-append_to_zshrc 'eval "$(rbenv init - --no-rehash)"' 1
+fancy_echo "Restarting services ..."
+brew_launchctl_restart 'postgresql'
+brew_launchctl_restart 'redis'
 
+fancy_echo "Relinking OpenSSL ..."
 brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
-brew_install_or_upgrade 'libyaml'
```

Comment:
I think this is related to SSL, and should stay here, though I'm not sure how it's related.